package.json

```diff
@@ -1,15 +1,17 @@
 {
-    "name": "@eclipse-cdt-cloud/vscode-ui-components",
+    "name": "@haydar-metin/vscode-ui-components",
```

Comment:
I assume the changes to referencing your fork need to be reverted.

Reply:
Reverted it back.